docs/excel/co-authoring-in-excel-add-ins.md

```diff
@@ -0,0 +1,48 @@
+# Coauthoring in Excel add-ins  
+
+With [coauthoring](https://support.office.com/en-US/article/Collaborate-on-Excel-workbooks-at-the-same-time-with-co-authoring-7152aa8b-b791-414c-a3bb-3024e46fb104), multiple people can work together and edit the same Excel workbook simultaneously. All coauthors of a workbook can see another coauthor’s changes as soon as that coauthor saves the workbook. To coauthor an Excel workbook, the workbook must be stored in OneDrive, OneDrive for Business, or SharePoint Online.
+
+> **Important:** In Excel 2016 for Office 365, you will notice AutoSave in the upper-left corner. When AutoSave is turned on, coauthors see your changes in real time. Consider the impact of this behavior on the design of your Excel add-in. Users can turn off AutoSave by via the switch in the upper left of the Excel window.
```

Comment:
"by via the switch" should be "via the switch"

Reply:
OK will fix